CHANGELOG.md

```diff
@@ -46,6 +46,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+### Fixes
+
+- [1680](https://github.com/umee-network/umee/pull/1680) Add amino support for MsgMaxWithdraw.
```

Comment:
is this a fix or a feature? I think the latter.

Reply:
Since the original code caused an unintended problem on the frontend, I'd call it a fix.